TLSharp.Core/Network/MtProtoPlainSender.cs

```diff
@@ -1,24 +1,27 @@
 ﻿using System;
 using System.IO;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace TLSharp.Core.Network
 {
     public class MtProtoPlainSender
     {
-        private int timeOffset;
-        private long lastMessageId;
-        private Random random;
-        private TcpTransport _transport;
+        private int _timeOffset;
+        private long _lastMessageId;
+        private readonly Random _random;
+        private readonly TcpTransport _transport;
```

Comment:
thanks for converting some fields to `readonly`! but this is not related to cancellation or deadlock fixing, so can you separate this into a different commit or even different PR?